src/user/user.service.ts

```diff
@@ -132,7 +132,12 @@ export class UserService {
       const rejected: StudentsImportJsonInterface[] = [];
 
       data.forEach(async (student) => {
```

Comment:
Całą tą funkcję webstorm podświetla mi na żółto i wyrzuca taki błąd:
Array<StudentImportFormatInterface>.forEach(     callbackfn: (value: StudentImportFormatInterface, index: number, array: StudentImportFormatInterface[]) => void,     thisArg?: any): void
The forEach() method executes a provided function once for each array element.